src/content/news/2021-04-20-serverless-operations-case-study.md

```
@@ -7,7 +7,7 @@ category: "お知らせ"
 
 クラウドサービスを使ったサーバーレス・アプリケーションの開発運用支援を手がける株式会社Serverless Operationsの利用事例に、Geoloniaが掲載されました。
 
-![](/images/news/2021-04-20-serverless-operations-case-study-1.png)
+![](/images/news/2021-04-20-serverless-operations-case-study-1.webp)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)